### ISSUE TITLE
homing doesn't behave as expected

### ISSUE THREAD

**Issue by fponticelli** [halofx/Marlin-2.0-Predator#2]
I use the TFT35 screen to interact with the board. When I send the `home` instruction the three motors do two small jumps in sequence instead of traveling to the endswitch. After I can move the motors correctly but they are limited by the fake homing performed.

I connected the endswitches to the X/Y/ZMax as per your `Configuration.h`.

**Comment by fponticelli** [comment]
After attempting many combos I got the homing to work. Unfortunately I am not sure what change made the difference :)

**Comment by halofx** [comment]
Is that the TFT35 v3, that also has the 12864 emulation? I have found the touch interface to be near worthless. I have been using the 12864 interface exclusively.

Not sure on the homing, but my leveling still isn't as consistent as I would like. With the TriGorilla board, I would occasionally get a bad level, but a second run would clear it up, and I could get perfect levels. I started with bi-linear, and now am using Unified bed leveling with better, but not perfect results. I think I might be gong too fast.
